source/index.js

```diff
@@ -121,6 +121,12 @@ module.exports = exports = function(params) {
         return res.send(result);
       });
   });
+  app.get(urn + '-vip', function(req, res) {
+    Promise.resolve(check(params))
```

Comment:
Yes it does 😄. I did not invent the Promise.resolve, I copied the one just above.